programs/drift/src/instructions/user.rs

```diff
@@ -1362,6 +1372,137 @@ pub fn handle_place_and_make_perp_order<'c: 'info, 'info>(
     Ok(())
 }
 
+pub fn place_swift_taker_order<'c: 'info, 'info>(
+    taker_key: Pubkey,
+    taker: &mut RefMut<User>,
+    taker_order_params_message: SwiftOrderParamsMessage,
+    ix_sysvar: &AccountInfo<'info>,
+    sig: [u8; 64],
+    perp_market_map: &PerpMarketMap,
+    spot_market_map: &SpotMarketMap,
+    oracle_map: &mut OracleMap,
+    state: &State,
+) -> Result<bool> {
```

Comment:
why does this return bool if it's not used by caller? is it for something in future?

Reply:
was from previous version where it was in one mega-instruction